### DISCUSSION TITLE
v1.3.0 - Settings and stuff

### DISCUSSION THREAD

**Discussion by sentenzo** [sentenzo/try-yt-dlp#10] [Announcements]
- the application now has a settings file (`tubic_settings.ini`)
- the settings window is available (the cog-icon in the top-right corner)
- now the user can:
  - choose the video size
  - choose the mp3-file quality
  - set the download folders
  - pick the browser to scan for the youtube cookies 
- tho "Open download folder" button (bottom-right corner)
- refactoring
- bugfix

<hr /><em>This discussion was created from the release <a href='https://github.com/sentenzo/tubic/releases/tag/v1.3.0'>v1.3.0 - Settings and stuff</a>.</em>

**Comment by oclever** [comment]
Thank you for this app! Very easy!!!
